File: app/src/androidTest/java/com/bopr/android/smailer/GoogleMailTest.java
package com.bopr.android.smailer;

import org.junit.Test;

import static com.google.api.services.gmail.GmailScopes.GMAIL_SEND;

public class GoogleMailTest  extends BaseTest{

    @Test
    public void send() throws Exception {
        MailMessage message = new MailMessage();
        message.setSubject("test");
        message.setBody("test");
        message.setRecipients("boprsoftdev@gmail.com");

        GoogleMail transport = new GoogleMail(getContext());
        transport.login("bo.smailer.service@gmail.com", GMAIL_SEND);
        transport.send(message);
    }
}

File: app/src/androidTest/kotlin/com/bopr/android/smailer/GoogleMailTest.kt
package com.bopr.android.smailer

import com.google.api.services.gmail.GmailScopes
import org.junit.Test

class GoogleMailTest : BaseTest() {

    @Test
    fun testSend() {
        val message = MailMessage(
                subject = "test",
                body = "test",
                recipients = "boprsoftdev@gmail.com"
        )
        val transport = GoogleMail(targetContext)
        transport.startSession("bo.smailer.service@gmail.com", GmailScopes.GMAIL_SEND)
        transport.send(message)
    }
}